### PR TITLE
Recyclerview divider

### PR DESCRIPTION
**Pull request by VEndymionV**
Line between items in RecyclerView

### CODE REVIEW

File: app/src/main/java/com/example/training1/view/MainActivity.kt [hunk 1]
```diff
@@ -40,9 +41,11 @@ class MainActivity : DaggerAppCompatActivity() {
     }
 
     private fun prepareRecyclerView() {
+        val itemsDivider: Drawable = applicationContext.getDrawable(R.drawable.line_divider)!!
```
Comment (awachowicz) [inline app/src/main/java/com/example/training1/view/MainActivity.kt]: I think applicationContext is not needed

File: app/src/main/java/com/example/training1/view/SimpleDividerItemDecoration.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import androidx.recyclerview.widget.RecyclerView
+
+class SimpleDividerItemDecoration(private val dividerDrawable: Drawable) :
+    RecyclerView.ItemDecoration() {
+
+    override fun onDrawOver(c: Canvas, parent: RecyclerView, state: RecyclerView.State) {
+        val left = parent.paddingLeft
+        val right = parent.width - parent.paddingRight
+
+        val childCount = parent.childCount
```
Comment (awachowicz) [inline app/src/main/java/com/example/training1/view/SimpleDividerItemDecoration.kt]: I think we should avoid iterating over children where it is possible.
Can you find better solution

File: app/src/main/java/com/example/training1/view/MainActivity.kt (hunk 1 above)
Comment (VEndymionV) [reply app/src/main/java/com/example/training1/view/MainActivity.kt]: You are right, I changed it in latest commit.

File: app/src/main/java/com/example/training1/view/SimpleDividerItemDecoration.kt (hunk 2 above)
Comment (VEndymionV) [reply app/src/main/java/com/example/training1/view/SimpleDividerItemDecoration.kt]: I created issue for this https://github.com/VEndymionV/Training1/issues/24

**Pull request merged**